### DISCUSSION TITLE
The openthread protocol stack ot1.3 is directly used to enable the function of the border router, and the external host cannot be pinged.

### DISCUSSION THREAD

**Discussion by tuopu** [openthread/openthread#8128] [Q&A] [answered]
Using the openthread protocol stack, when testing ot1.3, enable

"-DOPENTHREAD_CONFIG_BACKBONE_ROUTER_ENABLE=1"

"-DOPENTHREAD_CONFIG_NAT64_BORDER_ROUTING_ENABLE=1"

"-DOPENTHREAD_CONFIG_NAT64_TRANSLATOR_ENABLE=1"

"-DOPENTHREAD_CONFIG_MAC_CSL_RECEIVER_ENABLE=1"

"-DOPENTHREAD_CONFIG_DUA_ENABLE=1"

"-DOPENTHREAD_CONFIG_MLR_ENABLE=1"

Enter the br enable command, and the prompt is error 13: invalidstat. After viewing the program, it is found that BR has not been initialized. Which cli is used for BR initialization.

**Comment by jwhui** [comment, marked as answer]
The default OpenThread Border Router build should enable IPv6 routing. See the ot-br-posix CMake file for OpenThread: https://github.com/openthread/ot-br-posix/blob/main/third_party/openthread/CMakeLists.txt

I suggest taking a look at the [Thread Border Router - Bidirectional IPv6 Connectivity and DNS-Based Service Discovery](https://openthread.io/codelabs/openthread-border-router) code lab if you want to setup and play with Thread 1.3 features.

**Reply by tuopu** [reply]
Thank you for your guidance and reply.
